R/insertPackage.R

```diff
@@ -55,6 +55,7 @@
 ##' either \dQuote{gh-pages} indicating a branch of that name, or
 ##' \dQuote{docs/} directory in the main branch. The default value can
 ##' be overridden via the \dQuote{dratBranch} option.
+##' @param OSflavour an optional string naming the OSflavour, which is otherwise read as the second element of the 'Built' field of the \code{file}. For packages that do not need compilation on macOS for R >= 4.3 the 'Built' field is empty in the DESCRIPTION in a binary file (tgz), in which case it can be useful to set the \code{OSflavour} e.g. by the value of R.Version()$platform, so that \code{\link{insertPackages}} inserts the binary into the appropriate sub folder (under bin/maxosx).
```

Comment:
Can you please wrap the line as usual at about the same width as the other lines?  (My editor (Emacs) does that by itself in a markdown blob, others may too.)